app/routes/transaction.py

```diff
@@ -13,6 +19,28 @@ def transaction():
         else:
             transactions = Transaction.query.filter_by(merchant_id=current_user.id)
 
-        return render_template("transaction.html", transactions=transactions)
+        recentTransactions = []
+        for x in range(0, 4):
```

Comment:
I don't really get it ...

Reply:
C'est le truc pour séparer les transactions par journée. Regarde la classe au complet. @komlan55

Reply:
I have faith🙏